README.md

```diff
@@ -115,14 +104,31 @@ response.fail(String message)
 response.say("OK").send()
 ```
 
+## session
+```javascript
+// check if you can use session (read or write)
+Boolean session.isAvailable()
```

Comment:
Do you want to make a gettter in `request` object for getting `session` object so we don't need to pass `session` as a last argument to handlers?